ewasm-testnet-cpp-config.json

```diff
@@ -38,6 +38,7 @@
     "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" } },
     "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } } },
     "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "linear": { "base": 40000, "word": 0 } } },
-    "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" } }
+    "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" } },
+    "000000000000000000000000000000000000000a": { "code": "0x0061736d010000000113046000017f60037f7f7f0060027f7f0060000002430308657468657265756d0c63616c6c4461746153697a65000008657468657265756d0c63616c6c44617461436f7079000108657468657265756d0672657475726e000203020103070801046d61696e00030a18011601017f1000210041004100200010014100200010020b" }
```

Comment:
Needs to be updated to final version of the dummy code plus needs to have a balance of 1 wei.